hr_applicant/models/hr_app_med_occ.py

```diff
@@ -0,0 +1,113 @@
+from odoo import api, fields, models
+from odoo.tools.translate import _
+
+
+class ApplicantMedicalDetails(models.Model):
+    _name = "hr.applicant.medical.details"
+    _description = "Applicant Medical Details"
+    _rec_name = "medical_examination"
+
+    medical_examination = fields.Char("Medical Examination")
+    vital_sign = fields.Char("Vital sign")
+    date = fields.Date("Date", default=fields.Date.context_today, readonly=True)
+    doc_comment = fields.Char("Doctor’s Comments")
+
+    head_face_scalp = fields.Selection(
+        [("Abnormal", "Abnormal"), ("Normal", "Normal")], "Head, Face, Scalp"
+    )
+    nose_sinuses = fields.Selection(
+        [("Abnormal", "Abnormal"), ("Normal", "Normal")], "Nose/Sinuses"
+    )
+    mouth_throat = fields.Selection(
+        [("Abnormal", "Abnormal"), ("Normal", "Normal")], "Mouth/Throat"
+    )
+    ears_tms = fields.Selection(
+        [("Abnormal", "Abnormal"), ("Normal", "Normal")], "Ears/TMs"
+    )
+    eyes_pupils_ocular = fields.Selection(
+        [("Abnormal", "Abnormal"), ("Normal", "Normal")], "Eyes/Pupils/Ocular Motility"
+    )
+    heart_vascular_system = fields.Selection(
+        [("Abnormal", "Abnormal"), ("Normal", "Normal")], "Heart/Vascular System"
+    )
+    lungs = fields.Selection([("Abnormal", "Abnormal"), ("Normal", "Normal")], "Lungs")
+    abdomen_hernia = fields.Selection(
+        [("Abnormal", "Abnormal"), ("Normal", "Normal")], "Abdomen/Hernia"
+    )
+    msk_strengh = fields.Selection(
+        [("Abnormal", "Abnormal"), ("Normal", "Normal")], "MSK-Strength"
+    )
+    neurological = fields.Selection(
+        [("Abnormal", "Abnormal"), ("Normal", "Normal")],
```

Comment:
All selection has same value, in my opinion define global list and use same for all the fields.
cc @ShwetaSerpentCS